x-pack/plugins/observability/server/lib/rules/slo_burn_rate/executor.ts

```diff
@@ -53,7 +54,7 @@ async function evaluateWindow(slo: SLO, summaryClient: DefaultSLIClient, windowD
     toDurationUnit(windowDef.shortWindow.unit)
   );
 
-  const sliData = await summaryClient.fetchSLIDataFrom(slo, [
+  const sliData = await summaryClient.fetchSLIDataFrom(slo, ALL_VALUE, [
```

Comment:
What's the idea behind the burn rate rule and groupBy again? 
Are we gonna have one rule, and then, we'll compute the burn rate rule per (sloId, instanceId) and create an alert for that particular tuple?

Reply:
Correct.